src/lib.rs

```diff
@@ -855,12 +855,7 @@ impl Connection<Connected> {
     }
 
     #[inline]
-    fn write_pub_msg(
-        &self,
-        subj: &str,
-        reply: Option<&str>,
-        msgb: &[u8],
-    ) -> io::Result<()> {
+    fn write_pub_msg(&self, subj: &str, reply: Option<&str>, msgb: &[u8]) -> io::Result<()> {
```

Comment:
lol - ignore my other comment.. ;)

Reply:
This was actually the reason I checked in a `.rustfmt.toml` file, because I have a super aggressive personal config with a line width of 60 since I usually code with a few columns open to have as much related context directly in front of me as possible, and narrow columns are a bit faster for me to read personally.